src/iop/overexposed.c

```diff
@@ -185,10 +138,17 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
   const float *const restrict in = __builtin_assume_aligned((const float *const restrict)ivoid, 64);
   float *const restrict out = __builtin_assume_aligned((float *const restrict)ovoid, 64);
 
+  const dt_iop_order_iccprofile_info_t *const current_profile = dt_ioppr_get_pipe_current_profile_info(self, piece->pipe);
+  const dt_iop_order_iccprofile_info_t *const work_profile = dt_ioppr_get_histogram_profile_info(dev);
+
   // display mask using histogram profile as output
-  _transform_image_colorspace(self, in, img_tmp, roi_out);
+  // FIXME: the histogram already does this work -- use that data instead?
+  if(current_profile && work_profile)
+    dt_ioppr_transform_image_colorspace_rgb(in, img_tmp, roi_out->width, roi_out->height, current_profile,
+                                            work_profile, self->op);
+  else
+    fprintf(stderr, "[overexposed process] can't create transform profile\n");
```

Comment:
This keeps extant behavior, to warn and then do rest of work on uninitialized `img_tmp`. Is it better to:

```c
    dt_iop_copy_image_roi(ovoid, ivoid, ch, roi_in, roi_out, TRUE);
    return;
```

(as above when alloc fails) and warn to dt_control_log()? Could also copy `in` to `img_tmp`, and do the work in display profile -- and warn the user -- but that seems like too much trouble for uncertain results.

Reply:
Right, copying and emitting an error sounds good. Implemented now.